app/controllers/challenge_claims_controller.rb

```diff
@@ -1,142 +1,99 @@
 class ChallengeClaimsController < ApplicationController
-
   before_filter :users_only
-  before_filter :load_collection, :except => [:index]
-  before_filter :collection_owners_only, :except => [:index, :show, :create, :destroy]
-  before_filter :load_claim_from_id, :only => [:show, :destroy]
-
-  before_filter :load_challenge, :except => [:index]
-  
-  before_filter :allowed_to_destroy, :only => [:destroy]
-
+  before_filter :load_collection, except: [:index]
+  before_filter :collection_owners_only, except: [:index, :show, :create, :destroy]
+  before_filter :load_claim_from_id, only: [:show, :destroy]
+  before_filter :load_challenge, except: [:index]
+  before_filter :allowed_to_destroy, only: [:destroy]
 
   # PERMISSIONS AND STATUS CHECKING
 
   def load_challenge
     if @collection
       @challenge = @collection.challenge
     elsif @challenge_claim
-      @challenge = @challenge_claim.collection.challenge
+      # What we want is the ruby &. operator
```

Comment:
Maybe make this "TODO: Replace with &. operator after upgrading to Ruby 2.3" on the off chance we'll find it if we ever look at our TODO items.